Dockerfile

```
@@ -0,0 +1,33 @@
+
+# Install system dependencies and Poetry
+RUN apt-get update && apt-get install -y \
+    curl \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -sSL https://install.python-poetry.org | python3 - --version ${POETRY_VERSION}
+
+RUN mkdir /venv
+WORKDIR /venv
+COPY pyproject.toml .
+RUN poetry config virtualenvs.in-project true
+RUN poetry install --no-root
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)